tests/spanner_dbapi/test_types.py

```diff
@@ -52,7 +52,7 @@ def test_DateFromTicks(self):
         )
 
     def test_TimeFromTicks(self):
-        epochTicks = 1572851662.9782631  # Sun Nov 03 23:14:22 2019
+        epochTicks = 1572812062.9782631  # Sun Nov 03 23:14:22 2019
```

Comment:
@mf2199, I've got such a failure on Windows, and @tina80lvl reproduced it on Linux, so it looks like a solution.
Still, as it never failed on kokoro, we probably should check it with draft PR to be sure